openrtb-core/src/main/protobuf/openrtb-ext.proto

```diff
@@ -297,6 +303,28 @@ message SKAdNetworkFidelity {
   extensions 100 to 9999;
 }
 
+// SKAdNetwork SKOverlay attributes.
+message SKAdNetworkSKOverlay {
```

Comment:
Better to give this the same name as in the spec, i.e., `SKOverlay`, and scope it inside the `SKAdNetworkResponse` object.